### ISSUE TITLE
RAIL application window twitches after being dragged

### ISSUE THREAD

**Issue by Voxnod** [FreeRDP/FreeRDP#10661]
There is a couple of problems with RAIL application window moving in the x-client. 
The first one is that there is a some kind of window rectangle shadow staying at initial window location while the window is being dragged. 
The second is that when I finish dragging the window and release left mouse button the window seemingly wants to follow the cursor and twithcing. Sometimes in this twitching cursor location desynchronization occurs. This desync is cured by Alt-Tabbing out and in the window. I suppose some kind of redrawing process is initialize with window getting focused. 

Screen recording of the issues with some comments designed to show what I am doing:
https://github.com/user-attachments/assets/11191ef7-747d-4a16-834f-05c7b1d49729

I tried various gfx and hidef arguments. They did not help.

I am willing to help with this but cannot grasp where do I begin.

The command used to start the client:
```
xfreerdp /v:198.19.0.191 /u:user /p:pass /cert:ignore /app:program:"||Microsoft Word"
```
I am using FreeBSD 14.1 with Plasma 5. The RDP host is Windows 11.

**Comment by akallabeth** [comment]
@Voxnod yes, [rail](https://github.com/FreeRDP/FreeRDP/labels/channel%3Arail) does indeed have a lot of issues.
part of problem is that it requires quite tight integration with your window manager and the mapping of window flags is incomplete.
I did fix a bug quite recently with https://github.com/FreeRDP/FreeRDP/pull/10629 but that is certainly only a small part of your problems, but might give some pointers on where to look for issues regarding local window flag settings.

**Comment by Voxnod** [comment]
Thank you for the directions. I will try to get into it.

**Comment by stuaxo** [comment]
The Window shadow staying in one place (until the drag finishes) is visible in Microsoft word, but not in explorer.exe

Not sure if it's a side-effect of the widget set used.

Interestingly, Word under Wine has been doing a similar thing for many versions (I saw it in Word 2010 and 2016 for instance) - seperate shadow frame that stays at the initial position.
